CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
-# Contribute
+# Contribute to the documentation
+We really appreciate your support! Please have a look at the [issues page on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) and check if anybody already raised an issue with the same topic.
 
-Thanks for your support!
-Please open an issue on github. More to come soon...
-~~Please open a pull request on GitHub!~~
+If this is the case, please subscribe to the issue and explain your proposal.
+
+If that's not the case, please open a [new issue](https://github.com/jamulussoftware/jamuluswebsite/issues/new/choose), select which type of issue it is and describe your issue.
+
+If you want to open a Pull Request (after having described and discussed your proposal), have a look at the [Contribution page on the website](https://jamulus.io/wiki/Contribution). Keep in mind that your changes (if they are not just typos) might have to wait until everything is translated. We will bundle them (if everything is allright) with other changes and upload multiple changes in one bulk to jamulus.io.
+
+## I don't know GitHub. What do I need to know?
+If you aren't comfortable with Pull Requests, commits, etc. just open an issue. We will then care about your changes. If you want to do the changes on your own and open a Pull Request, follow this instructions:
```

Comment:
"follow **those** instructions"

Reply:
done, but what about "the following"?

Also "allright" (L8) doesn't look right.

Reply:
"Follow **these** instructions" if you're going to list them below. "Follow the following instructions" is a bit repetitive.

The preferred form is "all right", two words.